sky/backends/backend_utils.py

```diff
@@ -1958,11 +1958,10 @@ def _refresh_cluster_record(
 
     Args:
         cluster_name: The name of the cluster.
-        force_refresh: refresh the cluster status as long as the cluster exists.
-            If False, the cluster status will only be refreshed if the cluster:
+        force_refresh: if True, refresh the cluster status even if it may be skipped. Otherwise (the default),
```

Comment:
format (line too long)?

Reply:
Hmm, good catch! Weirdly, our format script and the GitHub CI failed to catch it. Fixed. Thanks!